client/src/site-search/index.tsx

```diff
@@ -55,50 +45,31 @@ export function SiteSearch() {
       // a client-side navigation happened.
       mountCounter.current++;
     }
-  }, [query.q, query.page, ga]);
+  }, [query, page, ga]);
 
   return (
     <div className="site-search">
       <PageContentContainer>
-        {query.q ? (
+        {query ? (
           <h1>
-            Results: {query.q}{" "}
-            {query.page && query.page !== "1" && (
-              <small className="current-page">Page {query.page}</small>
+            Results: {query}{" "}
+            {page && page !== "1" && (
+              <small className="current-page">Page {page}</small>
```

Comment:
As far as I know, `<small>` is a presentational element. Shall we still encourage usage of them? (Reminds me on HTML4 days).